db/schema.sql

```diff
@@ -5698,6 +5728,7 @@ WITH passed_block_number AS (
     FROM public.headers
     WHERE id = header_id
 )
+
```

Comment:
Would be cool to remove this from the diff if it's just whitespace. I think there's some difference in the way the schema gets generated bw mac vs linux :/